CDCgov/prime-reportstream
# 7556 [Permission] [API] Documentation

This PR adds documentation for the Permission related APIs in the application

Test Steps:
1. place docs/api/permissions.yml in an open api inspector (intellij has one built in)
2. we can now see something like this
<img width="1150" alt="Screenshot 2022-12-12 at 3 06 46 PM" src="https://user-images.githubusercontent.com/102491809/207180461-e83b063f-adb3-48fd-8622-459de60deb0d.png">


3. There is now documentation about the classes and DatabaseAccess apis that was added for the Permission (Feature Flag) feature
